pytest_dbt_adapter/builtin.py

```diff
@@ -219,11 +219,17 @@ def from_dict(cls, dct, overriding=None):
 
 
 TEST_PASSING_DATA_TEST = """
-select 1 as id where id = 2
+select * from (
+	select 1 as id
+) sbq
```

Comment:
why `sbq` ? what does it mean?
I'd rather go with a generic `t` .

Reply:
Short for `subquery`, could be anything. I just renamed to `my_subquery`